lib/js/i18n/i18next-scanner.config.js

```diff
@@ -54,7 +54,11 @@ module.exports = {
         metadata: {},
         allowDynamicKeys: false,
     },
-    input: ['src/**/*.{ts,tsx,js,jsx}'],
+    input: [
+        'src/**/*.{ts,tsx,js,jsx}',
+        '!../../lib/js/**/node_modules/**',
+        '../../lib/js/**/*.{ts,tsx,js,jsx}'
+    ],
```

Comment:
Why?

Reply:
They should be translated in the lib itself, then merged in the app with JS, see:
`databox/client/translations/index.ts`